### ISSUE TITLE
Suggestion - Lab 02a

### ISSUE THREAD

**Issue by TechGuyVincent** [MicrosoftLearning/AZ-104-MicrosoftAzureAdministrator#192]
# Module: 02
#### Step: 00

So I have had multiple users that have issues with this lab since sometimes they cant create a management group with the same name as the ID provided (az104-02-mg1). This seems to happen to users that use the Azure Pass Sponsorship more often then not. 

**My suggestion is that we change Task 2 Step 1**

From: 

{
   ],
   "AssignableScopes": [
       "/providers/Microsoft.Management/managementGroups/az104-02-mg1",
       "/subscriptions/SUBSCRIPTION_ID"
   ]
}

To: 

{
   ],
   "AssignableScopes": [
       "/providers/Microsoft.Management/managementGroups/*",
       "/subscriptions/SUBSCRIPTION_ID"
   ]
}

**And also on Step 5 on Cleanup Resources** 

From: 
_Remove-AzRoleAssignment -ObjectId '[object_ID]' -RoleDefinitionName 'Support Request Contributor (Custom)' -Scope $scope_

To: 
_Remove-AzRoleAssignment -RoleDefinitionName 'Support Request Contributor (Custom)' -Scope $scope_

Basically this would make the lab less sensitive to namechanges and subscription scopes and such. Just a suggestion. 

Regards, 

/V

**Comment by polichtm** [comment]
Hi Vincent,
could you clarify what exactly is the problem you're running into with the Management Groups in Azure Pass subscriptions? This works for me without any issues...

![image](https://user-images.githubusercontent.com/10284906/98879499-b0153f00-2453-11eb-9234-281eac74befa.png)

In addition, here is what I'm getting when attempting to run the command you suggested:

![image](https://user-images.githubusercontent.com/10284906/98880384-a1c82280-2455-11eb-92a5-13a3e1931ae0.png)

rgds
Marcin

**Comment by TechGuyVincent** [comment]
Hi again! 

I couldn't reproduce the issue on my own machine, but several people with the Azure Sponsorship did have the same issue (when using hotmail/outlook.com etc). 

If I read the docs page - it states: 

The Management Group ID is the directory unique identifier that is used to submit commands on this management group. **This identifier isn't editable after creation as it's used throughout the Azure system to identify this group**. The root management group is automatically created with an ID that is the Azure Active Directory ID. For all other management groups, assign a unique ID.

Could it be that only one unique ID is allowed to be created in Azure?

**Comment by polichtm** [comment]
Hi Vincent,
here is another subscription with the identically named management group:

![image](https://user-images.githubusercontent.com/10284906/99016646-81679900-2525-11eb-94a4-c6ccf30f3d75.png)

rgds
Marcin
